src/components/genre-view/genre-view.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import {Container, Card} from 'react-bootstrap';
+
+import './genre-view.scss';
+
+export class GenreView extends React.Component {
+  constructor() {
+    super();
+    this.state = {};
+  }
+
+  render() {
+    const {genre} = this.props;
+
```

Comment:
Have look at your state in the React add on for Chrome. You have `Genre` not `genre` 
![Screenshot 2021-01-16 at 12 10 36](https://user-images.githubusercontent.com/30231189/104810402-8d592b00-57f4-11eb-958e-6a8d270dcb7f.png)